src/chainparams.cpp

```diff
@@ -261,10 +261,10 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_ASSETS].bit = 5;
         consensus.vDeployments[Consensus::DEPLOYMENT_ASSETS].nStartTime = 1533924000; // GMT: Friday, August 10, 2018 6:00:00 PM
         consensus.vDeployments[Consensus::DEPLOYMENT_ASSETS].nTimeout = 1538351999; // GMT: Sunday, September 30, 2018 11:59:59 PM
-        consensus.vDeployments[Consensus::DEPLOYMENT_SCRYPT2].bit = 7;
-        consensus.vDeployments[Consensus::DEPLOYMENT_SCRYPT2].nStartTime = 1599037200;
-        consensus.vDeployments[Consensus::DEPLOYMENT_SCRYPT2].nTimeout = 1599073200;
-
+        
+        consensus.nNetworkPeriod[Consensus::NETWORK_PERIOD_X16R] = 0;

```

Comment:
I suppose testnet params should be the same with mainnet params

Reply:
testnet doesn't exist. Periods are set to begin earlier, so if someone will run the testnet he will reach the latest period faster.

Reply:
ok